docs/tutorials/hera_sim_defaults.ipynb

```diff
@@ -18,36 +18,15 @@
    "cell_type": "code",
```

Comment:
<p>Something got messed up here. Instead of plotting <code>np.polyval(...)</code>,we should just be plotting <code>bps[seas]</code> and <code>beams[seas]</code>,since they were evaluated a few cells up via <code>h1c_beam = hera_sim.defaults("omega_p")(fqs)</code> and so on. <code>hera_sim.defaults.("omega_p")</code> will return an instance of a subclass of <code>hera_sim.interpolators.FreqInterpolator</code> which may then be called to interpolate to the desired frequencies. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Totally! OK fixed that.</p>